src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpec.scala

```diff
@@ -215,9 +215,43 @@ class UserServiceSpec extends FlatSpec with Matchers with TestSupport with Mocki
     *      yes            skip    ---> User exists. Do nothing.
     */
   it should "return conflict when there's an existing subject for a given googleSubjectId" in{
-    val user = genCreateWorkbenchUserAPI.sample.get
+    val user = genCreateWorkbenchUser.sample.get
     dirDAO.createUser(WorkbenchUser(user.id, Some(user.googleSubjectId), user.email)).futureValue
     val res = service.registerUser(user).failed.futureValue
     res === new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.Conflict, s"user ${user} already exists")) shouldBe true
   }
+
+  "UserService inviteUser" should "create a new user" in{
+    val user = genInviteUser.sample.get
+    service.inviteUser(user).futureValue
+    val res = dirDAO.loadUser(user.inviteeId).futureValue
+    res shouldBe Some(WorkbenchUser(user.inviteeId, None, user.inviteeEmail))
+  }
+
+  it should "return conflict when there's an existing subject for a given userId" in{
+    val user = genInviteUser.sample.get
+    val email = genNonPetEmail.sample.get
+    dirDAO.createUser(WorkbenchUser(user.inviteeId, None, email)).futureValue
+    val res = service.inviteUser(user).failed.futureValue
+    res === new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.Conflict, s"identity with id ${user.inviteeId} already exists")) shouldBe true
```

Comment:
when testing for exception we usually use the intercept function. It is minor, but the test will Fail instead of Error if the exception is not caught.